meta-mender-core/recipes-bsp/u-boot/patches/0001-Add-missing-header-which-fails-on-recent-GCC.patch

```diff
@@ -1,14 +1,16 @@
-From d98fa92ea27a6a421310ebe1676c07d34c0c9bc8 Mon Sep 17 00:00:00 2001
+From 2a84a94f31981c00efcdbe4a907bdffd9601caf4 Mon Sep 17 00:00:00 2001
 From: Kristian Amlie <kristian.amlie@mender.io>
 Date: Mon, 15 Feb 2016 15:57:32 +0100
-Subject: [PATCH 1/4] Add missing header which fails on recent GCC.
+Subject: [PATCH] Add missing header which fails on recent GCC.
+
+Upstream-Status: Inappropriate
```

Comment:
Actually, this patch might be applicable. But I'm not even sure it's needed anymore, we could attempt to remove it.